database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
+  const businesses = [];
+  for (let i = 0; i < 100; i += 1) {
+    const entry = {
+      name: faker.company.companyName(),
+      claimed: Math.floor(Math.random() * Math.floor(2)),
+      // overallRating needs future refactor to generate dynamic rating
+      overallRating: Math.floor(Math.random() * Math.floor(5)) + 1,
+      // totalReviews needs future refactor to generate dynamic count
+      totalReviews: Math.floor(Math.random() * Math.floor(60)) + 10,
+      averageCost: Math.floor(Math.random() * Math.floor(4)) + 1,
+      businessType: `${faker.company.catchPhraseAdjective()}, ${faker.name.jobArea()}`,
```

Comment:
Cleaver combination of 2 faker functions

Reply:
!!